packages/gasket-nextjs/src/gasket-data-provider.js

```diff
@@ -6,8 +6,9 @@ export const GasketDataContext = React.createContext({});
 /**
  * Provider for the GasketData, adds context to child elements.
  *
- * @param {object} gasketData - Object of GasketData
- * @param {JSX.Element} children - Element to add GasketData context too
+ * @param {GasketData} props - Props
```

Comment:
Is `GasketData` the right data type here? Isn't that the type of just `gasketData`?

Reply:
The React does have a useful `PropsWithChildren<T>`  you can use, so you could have `@param {PropsWithChildren<{ gasketData: GasketData }>} props - Props`